CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 #### Enhancements
 
+* Make `unneeded_break_in_switch` auto correctable.
+  [KS1019](https://github.com/KS1019/)
+  [5188](https://github.com/realm/SwiftLint/pull/5188)
```

Comment:
```suggestion
* Make `unneeded_break_in_switch` auto correctable.  
  [KS1019](https://github.com/KS1019/)
```

Two spaces at the end are needed to enforce a line break. PRs are not mentioned, only issues that will be closed by this change.